src/app/table-module/table-module.module.ts

```diff
@@ -0,0 +1,37 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { HttpClient, HttpClientModule, HttpClientJsonpModule } from '@angular/common/http';
+import { ReactiveFormsModule } from '@angular/forms';
+
+import { GridModule } from '@progress/kendo-angular-grid';
+
+import { AppComponent } from './app.component';
+import { EditService } from './edit.service';
+
+
+
+const config: SocketIoConfig = { url: "http://localhost:5000", options: {} };
```

Comment:
move urls to env

Reply:
done

Reply:
once you move this to env. You have to get the URL from env.